samples/chemistry/LithiumHydrideGUI/README.md

```diff
@@ -4,7 +4,7 @@ This sample finds the equilibrium bonding distance of Lithium Hydride. Estimates
 
 ## Prerequisites
 
-On top of [.NET Core](https://www.microsoft.com/net/learn/get-started/macos),
+On top of [.NET Core](https://dotnet.microsoft.com/learn/dotnet/hello-world-tutorial/intro),
```

Comment:
I think we should change the text of this link from ".NET Core" to simply ".NET" since we have moved into .NET 6.0 already.

Reply:
Makes sense, I changed it!